### ISSUE TITLE
WS001: Incorrectly flags the title of the block

### ISSUE THREAD

**Issue by fergbrain** [docToolchain/asciidoc-linter#46]
The following is flagged as "warning":
```
.A mountain sunset
[#img-sunset,link=https://www.flickr.com/photos/javh/5448336655]
image::sunset.jpg[Sunset,200,100]


.Specify GitLab CI stages
[source,yaml]
----
image: node:16-buster
stages: [ init, verify, deploy ]
----
```

- Expected: `.` is treated as a block title (valid AsciiDoc syntax)
- Actual: asciidoc-linter flags it as “Missing space after the marker '.'” (WS001), treating it like a list item.

```
{
  "findings": [
    {
      "file": "bug-report-poc.adoc",
      "line": 1,
      "column": null,
      "message": "Missing space after the marker '.'",
      "severity": "warning",
      "rule_id": "WS001",
      "context": ".A mountain sunset"
    },
    {
      "file": "bug-report-poc.adoc",
      "line": 6,
      "column": null,
      "message": "Missing space after the marker '.'",
      "severity": "warning",
      "rule_id": "WS001",
      "context": ".Specify GitLab CI stages"
    }
  ]
}
```

References:
* https://docs.asciidoctor.org/asciidoc/latest/blocks/add-title/
* https://docs.asciidoctor.org/asciidoc/latest/macros/images/

**Comment by rdmueller** [comment]
thank you for filing this issue!

**Comment by fergbrain** [comment]
Everything seems to passing now, thanks! Still working on the report, so I'll file additional bugs as I find them.
